sycl/test-e2e/InvokeSimd/Regression/nbarrier_exec_in_order.cpp

```diff
@@ -108,8 +106,17 @@ ESIMD_INLINE void ESIMD_CALLEE_nbarrier(int localID,
 
 template <unsigned Threads, unsigned Size, bool UseSLM>
 [[intel::device_indirectly_callable]] SYCL_EXTERNAL void __regcall SIMD_CALLEE_nbarrier(
+#ifdef USE_ACC_PTR
```

Comment:
On the line 101 above is written:
```
  if constexpr (UseSLM) {
    auto res = experimental_esimd::lsc_slm_block_load<int, VL>(2 * off_slm);
    experimental_esimd::lsc_block_store<int, VL>(o + 2 * off, res);
  }
```
I maybe did not get the idea of the test, but that '2 *' part looks like an error. Perhaps, I am wrong. Shouldn't it be just 'off_slm' on L102 and 'o + off' on L103?

Reply:
This is the error, thanks!